design/use_cases/outdoor_navigation_use_case.md

```diff
@@ -0,0 +1,14 @@
+# Outdoor Navigation
```

Comment:
Rather than calling this "outdoor" -- I would suggest simply noting that we want non-planar navigation (that is, navigation on uneven ground).

Reply:
Why not outdoor? We have many use cases for outdoor delivery robots. Non-planarity may be a sub-requirement to this, but I'm not sure it captures the full intent. There may be other factors to consider for outdoor navigation besides planarity.

Reply:
If there are other requirements specific to outdoor, than that is fine name for the use case. As it it stands, there aren't any outdoor-specific requirements here, it was mainly >2D requirements.

Reply:
I want to capture more detailed requirements in the design/requirements folder. This is mainly intended to scope things at the top level. At this high level, we want the navigation to work outdoors as well as indoors.

Wait until we start talking about drone and submarine navigation! :)

Reply:
I agree with Matt, from a *robot user* standpoint the request can be to handle indoor navigation, outdoor, or both. The characteristics of each can be derived later.

Reply:
Actually we should be able to handle indoor and outdoor dynamically switching between the two as needed. That might be another use case to add.